upb/table.int.h

```diff
@@ -63,10 +63,14 @@ typedef struct {
 #endif
 
 /* Like strdup(), which isn't always available since it's not ANSI C. */
-char *upb_strdup(const char *s);
+char *upb_strdup(const char *s, upb_alloc *a);
```

Comment:
Nope, just table.c right now. reader.c has `upb_strndup()`. If either spreads beyond one file I'll move it to `upb.h`.